src/stateMachine.ts

```diff
@@ -107,6 +107,34 @@ export class StateMachine<
     });
   }
 
+  /**
+   * Generate a Mermaid StateDiagram of the current machine.
+   *
+   * Order your transitions so that the first and last are terminals
+   */
+  toMermaid( title?: string ) {
+    const diagram: string[] = [];
+    if (title) {
+      diagram.push("---");
+      diagram.push(`title: ${title}`);
+      diagram.push("---");
+    }
+    diagram.push("stateDiagram-v2");
+    diagram.push(`  [*] --> ${String(this.transitions[0].fromState)}`);
```

Comment:
trans 0 is not necessary the init state

Reply:
I realized this is not the case. My approach is very naive and relies on the user ordering their transitions.